README.md

```diff
@@ -33,12 +33,12 @@ build_katran.sh script. It should take care of all the required dependencies.
 If you need to build it for other linux distributions, yon need to make sure that
 1. it runs on recent linux kernel (4.13+)
 2. you have installed:
-  1. [folly](https://github.com/facebook/folly)
-  2. recent version of clang compiler (6.0+)
-  3. glog/gtest/gflags/elf libraries
-  4. [wangle](https://github.com/facebook/wangle)
-  5. recent version of [linux kernel src](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git)
-  6. if you want to build examples [fbthrift](https://github.com/facebook/fbthrift) and [grpc](https://github.com/grpc/grpc) must be installed as well
+	1. [folly](https://github.com/facebook/folly)
```

Comment:
Thanks for spotting this. few minor nits before we land it:
- could you please change from tabs to spaces and make indent equal to 2 (the rest of the codebase is using this formating)
- could you please switch from "rome nubmers" in nested list to something else (e.g. in nested list use "-" instead of numbers)

other then that everything looks great. thanks for helping us!